### PR TITLE
News week website clone with bootstrap 4

### PR DESCRIPTION
**Pull request by sinework**
This project is a clone of The News week website ( www.newsweek.com ).It's responsive to tablet view (breakpoint at 990px)

The webpage is built using html/css and bootstrap 4 classes and positioning technologies ( FLEX GRID FLOAT ).

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+    width: 49%;
+  }
+
+  .art-item-fst-img {
+    height: auto;
+  }
+
+  .font-wght img {
+    height: auto;
+  }
```
Comment (andresporras3423) [inline style.css]: Group selectors with the same style definition

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Sine Shiferaw & Khalil Hamdi** 
Good job with your first submission for this project :muscle:  :muscle:
Your page meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.
- [x]  There are some problems with the footer, especially in small screen size, let's checkout. First of all, the icons should be align below and not on the left side. Besides, the navbar should be fixed:

| Yours | Original |
|:---:|:---:|
|image | image |

- [ ] In the SUBSCRIBE section, small screen size, the text should be over the image, not on the left side, and the image should go upon the subscribe button:

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] In the NEWSLETTER section, small screen size, the image shouldn't appear and the other contain must be centered:

| Yours | Original |
|:---:|:---:|
|image | image |

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: style.css (hunk 1 above)
Comment (sinework) [reply style.css]: Thank you for your review. We have tried to fix every issue mentioned in the review.Let us know what you think.

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

### Hello @sinework and @khalilhamdii :wave: 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**